src/controllers/cart.controller.js

```diff
@@ -0,0 +1,36 @@
+import models from "./../database/models";
+
+export async function createCart(req, res) {
+    try {
+        const reqCart = req.body.cart;
+        const owner = req.params.ownerId;
+        console.log("\nowner\n", owner);
```

Comment:
console.log should not pushed to the version control

Reply:
I'm still working on it